src/app/components/alert/terra-alert-panel.component.ts

```diff
@@ -61,8 +66,9 @@ export class TerraAlertPanelComponent implements OnInit, OnDestroy
         window.removeEventListener(this.service.closeEvent, this.closeAlertListener);
     }
 
-    protected closeAlertByIndex(index:number):void
+    protected closeAlertByIndex(index:number, identifier:string):void
     {
+        this.closed.emit(identifier);
```

Comment:
Lieber ohne den zusätzlichen Parameter lösen. Die Methode heißt schließlich closeAlertByIndex
```suggestion
        const indentifier:string = this.alerts[index].identifier;
        this.closed.emit(identifier);
```

Reply:
mach ich